beps/0006-scaffolder-action-rollback/README.md

```diff
@@ -69,16 +225,25 @@ This section should describe the rollout process for any new features. It must t
 If there is any particular feedback to be gathered during the rollout, this should be described here as well.
 -->
 
+This feature enhancement will be optional, ensuring we maintain backwards compatibility.
+
+1. Update Type Definitions
+   - Add rollback to TemplateActionOptions and TemplateAction interfaces
+1. Ensure createTemplateAction Preserves Rollback
+   - Verify that the rollback function is properly passed through
+1. Modify TaskWorker
+   - Track completed actions during execution
+   - Implement rollback logic for failed tasks
+   - Handle rollback errors gracefully
+1. Update Documentation
+   - Describe the rollback feature
+   - Provide usage examples
+   - Explain best practices
+
 ## Dependencies
 
-<!--
-List any dependencies that this work has on other BEPs or features.
--->
+No known dependencies at this time.
```

Comment:
todo: are there any experimental features or scaffolder v2 details that could be dependencies?